src/plugins/MamicoCoupling.cpp

```diff
@@ -0,0 +1,80 @@
+#include "MamicoCoupling.h"
+#include "Domain.h"
+
+#ifdef MAMICO_COUPLING
+#include <coupling/interface/impl/ls1/LS1MamicoCouplingSwitch.h>
+//#include <LS1MamicoCouplingSwitch.h>
+#endif
+
+void MamicoCoupling::readXML(XMLfileUnits& xmlconfig)
+{	
+    return;
+}
+
+void MamicoCoupling::init(ParticleContainer* particleContainer,
+		DomainDecompBase* domainDecomp, Domain* domain)
+{
+    #ifdef MAMICO_COUPLING
+    if(_macroscopicCellService==NULL) 
+    {
+        _macroscopicCellService = (coupling::services::MacroscopicCellServiceImpl<ls1::LS1RegionWrapper,3>*)
```

Comment:
not sure what this line does without the coupling code :D Is the `*` operator overloaded?

Reply:
We are casting to that pointer type, the * is pointer

Reply:
lol I was probably confused by the long lines. Anyways don't do C-style casts but use proper C++ casts. I guess a pointer cast in that case?

Reply:
Will do